go.mod

```diff
@@ -242,7 +244,7 @@ require (
 	go.uber.org/atomic v1.11.0
 	go.uber.org/automaxprocs v1.5.3
 	go.uber.org/dig v1.17.0
-	go.uber.org/fx v1.18.2
+	go.uber.org/fx v1.20.1
```

Comment:
Don't update fx. Make sure to set fx version  `v1.18.2` in all modules. There are some known issued with the upgrade